src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,33 @@
+.stars {
+  display: flex;
+  margin-top: 16px;
+  align-items: flex-end;
+  text-align: justify;
```

Comment:
The `text-align: justify;` property is not typically used with flexbox layouts. Consider removing it or using a more appropriate property for aligning flex items.